README.md

```diff
@@ -311,59 +361,3 @@ python examples/ckan-upload-csv.py
 
 ## Integrations
 
-### Using Google Cloud Composer
```

Comment:
This is quite a bit of useful info we are cutting. Don't we want to keep it somewhere?

Reply:
@hannelita ^^^

Reply:
This section became `ckan_api_load_gcp DAG: Using Google Cloud Composer` - there is a DAG to handle GCP